src/api/events.js

```diff
@@ -1,5 +1,4 @@
 import matches from '../util/matches-selector';
```

Comment:
Do we like the rename from singular to plural? I find that it better describes the intent of the function that it supports many events rather than one event.